modules/oadp-review-backup-restore.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oadp-review-backup-restore_{context}"]
+= Preview resources before running backup and restore
+
+OADP backs up application resources by type, namespace, or label at runtime. This means, you are able to view the resources after the backup is complete. Similarly, you are able to view the restored objects by namespace, PV, or label after a restore operation is complete. You can do a dry run of the backup and restore operations, to preview the resources in advance.
+
+To preview the resources that will be backed up before running the actual backup, run the following `velero` command:
+
+[source, terminal]
+----
+velero backup create <backup-name> --snapshot-volumes false # <1>
+----
+<1> Specify `--snapshot-volumes false`.
+
+To preview the resources that will be restored before running the actual restore, run the following `velero` command:
+
+[source, terminal]
+----
+velero restore create --from-backup <backup-name> # <1>
```

Comment:
@weshayutin @anarnold97 @shdeshpa07 Above and this command will create a actual restore in the cluster.  Maybe we will have to tell user to delete all the resources created by this restore after reviewing the resources.

Reply:
Hi @PrasadJoshi12 - added a note as mentioned in review comment. Please review. Thanks.